app/src/main/java/com/kylecombes/micrepeater/Setting.java

```diff
@@ -0,0 +1,44 @@
+package com.kylecombes.micrepeater;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.util.DisplayMetrics;
+import android.view.View;
+import android.widget.CompoundButton;
+import android.widget.ImageButton;
+import android.widget.Switch;
+
+
+public class Setting extends Activity {

```

Comment:
Change the class name `Setting` to `SettingActivity`. You can do that by moving your cursor to the word Setting, using `Shift + F6` (on Windows) to rename it. Android Studio will automatically find all usages and rename it for you.